composer.json

```diff
@@ -21,7 +25,7 @@
         "symfony/options-resolver": "~2.6 || ~3.0"
     },
     "require-dev": {
-        "phpspec/phpspec": "2.0.*@dev"
+        "phpspec/phpspec": "~2.4.0"
```

Comment:
This constraint forbids using 2.5, which is a bad idea IMO. the constraint I submitted in #28 is better IMO